app/assets/stylesheets/application.scss

```diff
@@ -17,19 +13,20 @@ a {
   overflow: auto;
 }
 
-.form-input {
-  margin-bottom: 30px;
+.banner h1 {
+  font-size: 2em;
 }
 
-.csv {
-  border-collapse: collapse;
-  table-border: 1;
-}
-
-.csv, .csv th, .csv td {
-  border: 1px solid black;
-  text-align: left;
+footer {
+  margin-top: 200px;
```

Comment:
Properties should be ordered background-color, border-top-left-radius, border-top-right-radius, bottom, color, font-size, margin-top, padding, text-align